src/Uri.js

```diff
@@ -281,6 +280,13 @@ class Uri {
 					break;
 				default:
 					url += opt_uri;
+
+					var uriScheme = opt_uri.substr(0, opt_uri.indexOf(':'));
+					if (uriScheme &&
+						['.', 'localhost'].every((value) => uriScheme.indexOf(value) === -1)) { // jshint ignore:line
```

Comment:
@fernandosouza,

Thanks for the catch on that!  I must have missed those.  

I'll make the changes and force push it.